packages/feedback/src/FeedbackForm.js

```diff
@@ -210,11 +210,11 @@ const FeedbackForm = ({
             <Button
               onClick={onClose}
               color="secondary"
-              onKeyDown={({ keyCode }) => {
-                if (keyCode === 13) {
+              onKeyDown={(e) => {
```

Comment:
instead of switching away from destructuring here, I think we should add `shiftKey` to the args.
`({ keyCode, shiftKey }) =>`